docs/ref/models/instances.txt

```diff
@@ -355,10 +355,17 @@ To save an object back to the database, call ``save()``:
 
 .. method:: Model.save(force_insert=False, force_update=False, using=DEFAULT_DB_ALIAS, update_fields=None)
 
+You can view the following sections for a more detailed information each
+argument to this method.
+
+    - :ref:`force_insert <ref-models-force-insert>`.
+    - :ref:`force_update <ref-models-force-insert>`.
+    - :ref:`update-fields <ref-models-update-fields>`.
+
 If you want customized saving behavior, you can override this ``save()``
 method. See :ref:`overriding-model-methods` for more details.
 
-The model save process also has some subtleties; see the sections below.
+The model save process also has some subtleties, see the sections below.
```

Comment:
This is a separate, and in any case, the semicolon seems more right to me, IMO (it is a compound sentence).

Reply:
Yes, it's not a typo. Please revert.

Reply:
reverted.